Content.Server/GameObjects/Components/Mobs/SpeciesComponent.cs

```diff
@@ -6,17 +6,22 @@
 using Content.Shared.GameObjects;
 using Content.Shared.GameObjects.Components.Mobs;
 using Robust.Server.GameObjects;
-using Robust.Shared.ContentPack;
+using Robust.Shared.Audio;
 using Robust.Shared.GameObjects;
 using Robust.Shared.Interfaces.GameObjects;
 using Robust.Shared.Interfaces.Network;
 using Robust.Shared.Serialization;
+using Robust.Shared.IoC;
+using Robust.Server.GameObjects.EntitySystems;
 
 namespace Content.Server.GameObjects
 {
     [RegisterComponent]
     public class SpeciesComponent : SharedSpeciesComponent, IActionBlocker, IOnDamageBehavior, IExAct
     {
+
+        [Dependency] private readonly IEntitySystemManager _entitySystemManager;
```

Comment:
You should probably disable warning CS0649 (Field never assigned to) when using IoC dependencies.
Like this:
```csharp
#pragma warning disable 649
    [Dependency] private readonly IEntitySystemManager _entitySystemManager;
#pragma warning restore 649
```